app/src/sharedTest/java/org/oppia/app/options/OptionsFragmentTest.kt

```diff
@@ -117,26 +122,44 @@ class OptionsFragmentTest {
       )
       onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(10))
       onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_text_view)).check(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0, R.id.story_text_size_text_view
+        )
+      ).check(
         matches(withText("Large"))
       )
     }
   }
 
+  /* ktlint-disable max-line-length */
   @Test
   fun testOptionFragment_clickStoryTextSize_changeTextSizeToLarge_changeConfiguration_checkTextSizeLargeIsSelected() {
     launch<OptionsActivity>(createOptionActivityIntent(0)).use {
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_item_layout)).perform(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0,
+          R.id.story_text_size_item_layout
+        )
+      ).perform(
         click()
       )
       onView(withId(R.id.story_text_size_seekBar)).perform(clickSeekBar(10))
       onView(isRoot()).perform(orientationLandscape())
       onView(withContentDescription(R.string.go_to_previous_page)).perform(click())
-      onView(atPositionOnView(R.id.options_recyclerview, 0, R.id.story_text_size_text_view)).check(
+      onView(
+        atPositionOnView(
+          R.id.options_recyclerview,
+          0, R.id.story_text_size_text_view
+        )
+      ).check(
         matches(withText("Large"))
       )
     }
   }
+  /* ktlint-enable max-line-length */
```

Comment:
Same comments from the other PR holds here, we should only disable the check for the function name, not the entire block. Also please try `// ktlint-disable max-length-line` right next to the function name. I think that should work instead of making block rules

Reply:
Done

Reply:
Resolving this on behalf of @vinitamurthi . Though it was not mentioned by her, but as the changes are are fairly simple and this PR has been approved below by @vinitamurthi.